lib/db/util.js

```diff
@@ -37,8 +37,24 @@ const VERIFICATION_METHODS = new Map([
   ['totp-2fa', 2]   // TOTP code
 ])
 
+// If you modify one of these maps, modify the other.
```

Comment:
This is fine for now but I think, if the capabilities list gets longer in the future, we should definitely switch to generating one of these programmatically so that there's only one definition to maintain.